### ISSUE TITLE
accordion Bottom border is zero by default

### ISSUE THREAD

**Issue by dgpuranik** [ng-bootstrap/ng-bootstrap#2338]
### Bug description:
I am using simple Accordion form your package. I noticed bottom border for accordion is by default 0. All nested elements like cards inherit the same property so they also dont show bottom border

My HTML looks like this 

```
<div class="mt-4">
<ngb-accordion #acc="ngbAccordion">
  <ngb-panel title="{{category}}" *ngFor="let category of categories">
    <ng-template ngbPanelContent>
      <div class="container">
        <div class="row">
          <div class="col-md-4" *ngFor="let menuItem of menuItems">
            <div class="card mb-5 box-shadow">
              <img class="card-img-top" [src]="images[0]" alt="Card image cap">
              <div class="card-body">
                <p class="card-text">This is a wider card with supporting text below.</p>
                <div class="d-flex justify-content-between align-items-center">
                  <div class="input-group">
                    <span class="input-group-btn">
                        <button type="button" class="btn btn-plus-minus btn-number" (click)="onMinusClick(menuItem)">
                            <i class="fa fa-minus"></i>
                        </button>
                    </span>
                    <input [(ngModel)]="menuItem.quantity" class="form-control input-number" value="0" min="0" max="100">
                    <span class="input-group-btn">
                        <button type="button" class="btn btn-plus-minus btn-number" (click)="onPlusClick(menuItem)">
                            <i class="fa fa-plus"></i>
                        </button>
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </ng-template>
  </ngb-panel>
</ngb-accordion>
</div>

<div class="mt-4 mb-4 text-center">
  <button class="btn btn-secondary" (click)="checkout()">Checkout</button>
</div>
```

When I load the page, the following pic shows the initial state

<img width="957" alt="card-bottom-border-absent" src="https://user-images.githubusercontent.com/3536670/39095432-622003a8-4651-11e8-9742-4edc164f52e6.PNG">

I tried to some tricks in Chrome debugger and found one way to get those absent borders back. Its visible in pic:

<img width="956" alt="card-bottom-border-present" src="https://user-images.githubusercontent.com/3536670/39095446-8ffa8834-4651-11e8-9349-341571e2487f.PNG">

Angular: 5.2.9

ng-bootstrap: 1.1.2

Bootstrap: 4.1.0

**Comment by dgpuranik** [comment]
I understand the logic behind the bottom border to be 0 except last one if there are multiple collapses. But when there is only one collapse then bottom border should not be zero. Cards put inside the accordion inherits this property and I didnt find any way to stop that inheritance.

**Comment by pkozlowski-opensource** [comment]
This is, most likely, issue in the Bootstrap CSS and should be reported there. But I can't be sure for 100% since you didn't share any reproduce scenario nor crucial info like the library version, Bootstrap CSS version etc.

Please follow our issue template when reporting bugs: https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug
It is very hard / impossible to help without enough context.

**Comment by dgpuranik** [comment]
I have provided necessary versions.  

I am trying to create plunker so you will have better idea.

**Comment by dgpuranik** [comment]
I couldnt reproduce the issue in plunker.  This is the link http://plnkr.co/edit/bt1GEWBZXIMHSjgHZ0Bl . I copied the mark up in my app code and issue appeared again.

I am really not sure but its nothing to do with your library.

**Comment by pkozlowski-opensource** [comment]
You need to use Bootstrap CSS 4.1 to reproduce: http://plnkr.co/edit/sqVvQspnaM8N5vf4DNG0?p=preview

If you inspect the rule which removes bottom border it is this one: https://github.com/twbs/bootstrap/blob/9faf8e85451ebc5981e85bd22d305f5275a4fd24/scss/_card.scss#L279-L301

It seems like Bootstrap CSS is not handling cards within accordions properly.

**Comment by dgpuranik** [comment]
Will reverting back to 4.0 help? How can I take this further so its fixed?

**Comment by pkozlowski-opensource** [comment]
> Will reverting back to 4.0 help?

It should for this particular issue.

> How can I take this further so its fixed?

It needs to be fixed in the Bootstrap CSS so I would suggesting opening an issue in https://github.com/twbs/bootstrap/issues

I'm going to close this one here as there is nothing on our side that we could fix.

**Comment by svenevs** [comment]
If anybody happens to find this issue, it will be released in bootstrap 4.3, [patch fixing it is here](https://github.com/twbs/bootstrap/pull/27881).

**Comment by mythcyrax** [comment]
This was not solved in 4.3. Bootstrap.css still contains 
```
.accordion > .card:first-of-type {
  border-bottom: 0;
```

**Comment by kampalex** [comment]
> This was not solved in 4.3. Bootstrap.css still contains
> 
> ```
> .accordion > .card:first-of-type {
>   border-bottom: 0;
> ```

Looks like a single card in an accordion. I ran also into this issue.
I've create a [pull request](https://github.com/twbs/bootstrap/pull/28780) for preventing border-bottom in this situation.
ps. HTML in original post must be rebuild to contain .card children directly under .accordion.

**Comment by JrmSK** [comment]
Found a solution on my own.
I was facing this issue, coupled with an other challenge: on the same page i am displaying a few different accordions, some might be 1 item long, some more. So I had to find a trick to select only the ones with one child. I got help from this article: https://css-tricks.com/almanac/selectors/o/only-child/

the rest is pretty straightforward, you just need to add a css rule:

`.accordion div.card:only-child {
  border-bottom: 1px solid rgba(0, 0, 0, 0.125);
  border-radius: calc(0.25rem - 1px);
}
`
